### PR TITLE
[JENKINS-26186] Enable changing display format for Last Build Trigger Column

### PR DESCRIPTION
**Pull request by ljader**
Users can now pick how to display build Cause in mentioned column,
since not all users want to see long descriptions.

By default column will display only icon for build cause.

### CODE REVIEW

**Comment by ljader** [comment]
@batmat and @mpapo please review this in free time :)

**Comment by Batmat** [comment]
Yup, ack. Will look into it asap.

Cheers

2015-08-18 7:21 GMT+02:00 Łukasz Jąder <notifications@github.com>:

> @batmat <https://github.com/batmat> and @mpapo <https://github.com/mpapo>
> please review this in free time :)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jenkinsci/buildtriggerbadge-plugin/pull/15#issuecomment-132073893>
> .
>



-- 
Baptiste <Batmat> MATHUS - http://batmat.net
Sauvez un arbre,
Mangez un castor !

**Comment by Batmat** [comment]
Code seems fine. I'll try to find some time in the next few days to at least try the modification locally before merging. 

That may be occasion for PR16 to be adjusted in the meantime. 

And I'll issue a release in the same go.

Thanks again @ljader !

File: src/main/java/org/jenkinsci/plugins/buildtriggerbadge/LastBuildTriggerColumn.java [hunk 1]
```diff
@@ -24,6 +25,25 @@
  * 
  */
 public class LastBuildTriggerColumn extends ListViewColumn {
+	public static final String ICON_ONLY = "iconOnly";
+	public static final String ICON_AND_DESC = "iconAndDesc";
```
Comment (Batmat) [inline src/main/java/org/jenkinsci/plugins/buildtriggerbadge/LastBuildTriggerColumn.java]: IMO using an enum here would have been better, but not sure Jenkins's automatic conversion would then work correctly.

**Pull request merged**